scripts/resources.js

```diff
@@ -0,0 +1,297 @@
+const $ = require('shelljs')
```

Comment:
about the shebang, it does not work in win32 environments (without cygwin / mingw), so
i suggest to run the script using node from command line (which is cross platform)